fog/ledger/server/tests/connection.rs

```diff
@@ -658,7 +743,36 @@ fn add_block_to_ledger_db(
     let num_blocks = ledger_db.num_blocks().expect("failed to get block height");
 
     let new_block;
+
+    // Get a random timestamp
```

Comment:
Nit: The timestamp is not random, it is derived from the number of blocks.

Reply:
I made the change to the comment